### PR TITLE
Added metadata to examples (#11765)

### PR DESCRIPTION
**Pull request by Christophers27**
Added metadata to examples/basic/lines/line_single.py
Partially addresses issue: #11765

### CODE REVIEW

File: examples/basic/lines/line_single.py [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+   between defined points.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.line
+    :refs: :ref:`ug_basic_lines_single`
+    :keywords: line, glyph, straight
+
+..
+
+'''
```
Comment (bryevdv) [inline examples/basic/lines/line_single.py]:
```suggestion

'''
```

File: examples/basic/lines/line_single.py [hunk 2]
```diff
@@ -1,3 +1,15 @@
+'''A line graph using user-defined data points. This example demonstrates
+   the use of the ``line`` function to make a graph by mapping straight lines
+   between defined points.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.line
+    :refs: :ref:`ug_basic_lines_single`
+    :keywords: line, glyph, straight
```
Comment (bryevdv) [inline examples/basic/lines/line_single.py]: We usually stick to nouns/concepts for keywords 
```suggestion
    :keywords: line, glyph
```

**Review by bryevdv — CHANGES_REQUESTED** [review]
Hi @Christophers27 thanks for the PR! Just a couple of minor changes

**Comment by Christophers27** [comment]
Hi @bryevdv I committed the changes you asked but it seems that it caused some errors? Sorry I haven't been able to figure out why after some time.

**Pull request closed without merging**